netmiko/ssh_dispatcher.py

```diff
@@ -31,7 +31,6 @@
 from netmiko.dell import DellPowerConnectSSH
 from netmiko.dell import DellPowerConnectTelnet
 from netmiko.dell import DellIsilonSSH
-from netmiko.dlink import DlinkDSTelnet, DlinkDSSSH
 from netmiko.eltex import EltexSSH, EltexEsrSSH
```

Comment:
It looks like you need to `git rebase` as you are removing a few things you shouldn't be removing.

Reply:
Please check my new commit, should of fixed it